examples/requirements.txt

```diff
@@ -1,3 +1,3 @@
 transformers~=4.27.1
-diffusers==0.11.1
+diffusers==0.14.0
```

Comment:
Do diffusion models work for you after (de)serializing with diffusers 0.14.0?

After creating a fresh conda env with python 3.8 and pip installing the requirements, I run into a couple of issues trying to run this script.
`python hf_serialization.py stabilityai/stable-diffusion-2 ./data --model_type=diffusers --validate`
1. You also need scipy installed so it should be added to the requirements.txt:
```
ImportError: 
LMSDiscreteScheduler requires the scipy library but it was not found in your environment. You can install it with pip: `pip install
scipy`
```
2. The validation image produced after serializing/deserializing SD2 is messed up:
![test](https://user-images.githubusercontent.com/24724407/227346030-61e308b4-345f-4708-bce9-e7d6da5b2fb5.png)